documentation/docs/evaluators/general_evaluator.md

```diff
@@ -1 +1,8 @@
-# General evaluator
\ No newline at end of file
+# General evaluator
+The general runner is capable of running the standard library of the following languages
```

Comment:
running the following languages with standard libraries included

Reply:
https://github.com/SELab-2/UGent-3/pull/424/commits/707631c951bb7f16d4e73d19aba62882d8f974ed